test/cppsuite/src/main/test.h

```diff
@@ -70,15 +63,6 @@ class test : public database_operation {
      */
     virtual void run();
 
-    /*
```

Comment:
I don't see any function definitions for these. Are we cleaning up junk code here?

Reply:
Yes, correct, there were never implemented surprisingly.